features/foutafhandeling.feature

```diff
@@ -5,141 +5,65 @@ Functionaliteit: Afhandeling van fouten
     Wil ik consistente en gebruiksvriendelijke fout responses conform problem+json
     Zodat ik in mijn consumer applicatie de fouten eenduidig kan afhandelen.
 
-Rule: verplichte parameter(s) zijn opgegeven
-
-    Scenario: Verplichte parameter is niet opgegeven
-        Als '/adressen/zoek' wordt aangeroepen
-        Dan bevat de response de volgende velden
-        | naam   | waarde                                                                            |
-        | title  | Ten minste één parameter moet worden opgegeven.                                   |
-        | status | 400                                                                               |
-        | code   | paramsRequired                                                                    |
-        | detail | Required request parameter 'zoek' for method parameter type String is not present |
-        En bevat de response geen invalidParams
-
-Rule: alleen gespecificeerde parameters mogen worden gebruikt
+Rule: waarde van parameter is valide
```

Comment:
of is de regel zoiets als "Als een parameterwaarde niet valide is wordt een foutmelding gegeven"? of "de waarde van een parameter moet valide zijn"?